RecoTracker/LST/BuildFile.xml

```diff
@@ -3,6 +3,7 @@
 <use name="lst_headers"/>
 <use name="DataFormats/TrackerRecHit2D"/>
 <use name="HeterogeneousCore/AlpakaInterface"/>
+<flags CXXFLAGS="-DLST_STANDALONE"/>
```

Comment:
now that it's explicitly here, I'm thinking if `LST_EXTERNAL` is a more appropriate name to avoid confusion in the purpose.
Since either `EXTERNAL` or `STANDALONE` are technically to indicate a negative of being in CMSSW, perhaps even better is to use smth like `LST_IS_CMSSW_PACKAGE`

Reply:
Yeah, I think something like `LST_IS_CMSSW_PACKAGE` makes sense. I'll close this PR and make the required changes to the other PR.